features/bundle/index.js

```diff
@@ -13,8 +13,14 @@ function packageFeature({ archiveName, moduleName, artifactPath }, targetDir) {
         moduleName,
         artifactPath
     );
-    const args = ["-r", targetArchive, "."];
-    child_process.execFileSync("zip", args, { cwd: sourceDir });
+    const args = windowsEnvironment
+        ? ["-cfM", targetArchive, "."]
+        : ["-r", targetArchive, "."];
+    if (windowsEnvironment) {
+        child_process.execFileSync("jar", args, { cwd: sourceDir });
+    } else {
+        child_process.execFileSync("zip", args, { cwd: sourceDir });
+    }
```

Comment:
Hm, so I guess you could make this shorter by conditionally setting some `cmd` variable similarly to how you did it in `build.js`.

But I wonder: The reason why we have `zip` on our systems is because we made sure it's there. You just happen to have `jar` on your system, which happens to be able to do roughly the same as `zip`. But someone who doesn't have the Android and/or Java SDK probably doesn't have `jar` either. Even with the existing logic - we have a hidden dependency, and now it hurts us (rightly so) :D

I think the clean approach here would be to use some NPM package for zipping the files, although I'd be OK with tackling that after this PR lands. At least a chonky TODO comment would be nice though.